vcr/cassette.py

```diff
@@ -87,16 +88,21 @@ def _execute_function(self, function, args, kwargs):
         return handler(function, args, kwargs)
 
     def _handle_coroutine(self, function, args, kwargs):
+        """Wraps a bi-directional generator so that we're inside the cassette
```

Comment:
nitpick but
bi-directional generator == coroutine, right?

I'd prefer to use the typical terminolgy here.